src/explore-education-statistics-admin/src/modules/find-statistics/components/EditableContentSubBlockRenderer.tsx

```diff
@@ -52,7 +60,7 @@ class EditableContentSubBlockRenderer extends Component<Props> {
   public render() {
     const { block } = this.props;
 
-    return EditableContentSubBlockRenderer.getBlockHTML(block);
+    return this.getBlockHTML(block);
```

Comment:
This method feels like it could be inlined here

Reply:
I guess at the moment it can. There was a requirement to drag the content within the blocks around, but this hasn't been re-requested at the moment.

It can be changed later if required, and it probably will be. But given the change in pattern for how it's done it may be better to start anew anyway.